src/main/external-player.js

```diff
@@ -17,12 +17,12 @@ let proc = null
 function checkInstall (playerPath, cb) {
   // check for VLC if external player has not been specified by the user
   // otherwise assume the player is installed
-  if (playerPath == null) return vlcCommand(cb)
+  if (playerPath.length === 0) return vlcCommand(cb)
```

Comment:
Won't this throw a TypeError when `playerPath` _is_ `null` tho?

Reply:
That's why I included a reference to the migration file replacing null by an empty string.
However if it still possible to receive null I will modify my PR accordingly.